transform/snowflake-dbt/models/events/user_events_telemetry.sql

```diff
@@ -1,7 +1,7 @@
 {{config({
     "materialized": "incremental",
```

Comment:
@sri-byte , does it make sense to keep having this config setting here since we don't seem to be using it?